dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -163,8 +169,20 @@ public static void main(String[] arguments)
         DependencyGraphBuilder.getStaticLinkageCheckDependencies(artifacts);
     List<DependencyPath> dependencyPaths = dependencyGraph.list();
 
+    // To remove duplicates on (groupId:artifactId) for dependency mediation
+    Set<String> artifactsInPaths = Sets.newHashSet();
+
     for (DependencyPath dependencyPath : dependencyPaths) {
       Artifact artifact = dependencyPath.getLeaf();
+      // groupId:artifactId
+      String dependencyMediationKey = Artifacts.makeKey(artifact);
+      if (!artifactsInPaths.add(dependencyMediationKey)) {
```

Comment:
Do we still need a multimap here if we're only adding each groupId:aritfactId the first time?